icu4c/source/i18n/unicode/measunit.h

```diff
@@ -52,31 +52,32 @@ class LongNameHandler;
  * The complexity determines which operations are available. For example, you cannot set the power
  * or prefix of a compound unit.
  *
- * @draft ICU 67
+ * @stable ICU 67
  */
 enum UMeasureUnitComplexity {
     /**
      * A single unit, like kilojoule.
      *
-     * @draft ICU 67
+     * @stable ICU 67
      */
     UMEASURE_UNIT_SINGLE,
 
     /**
      * A compound unit, like meter-per-second.
      *
-     * @draft ICU 67
+     * @stable ICU 67
      */
     UMEASURE_UNIT_COMPOUND,
 
     /**
      * A mixed unit, like hour+minute.
      *
-     * @draft ICU 67
+     * @stable ICU 67
      */
     UMEASURE_UNIT_MIXED
 };
 
+#ifndef U_HIDE_DRAFT_API
```

Comment:
looks like line 41 was un-removed and needs to be re-removed

Reply:
Thank you; I'm an idiot.

Reply:
Now I just need to figure out how to do that...